internals/secrethub/generate.go

```diff
@@ -59,13 +60,31 @@ func (cmd *GenerateSecretCommand) Register(r Registerer) {
 }
 
 // before configures the command using the flag values.
-func (cmd *GenerateSecretCommand) before() {
-	cmd.generator = randchar.NewGenerator(cmd.useSymbols)
+func (cmd *GenerateSecretCommand) before() error {
+	useSymbols := false
+
+	useSymbolsEnv := os.Getenv("SECRETHUB_GENERATE_RAND_SYMBOLS")
+	if useSymbolsEnv != "" {
+		b, err := strconv.ParseBool(useSymbolsEnv)
+		if err != nil {
+			return err
+		}
+		useSymbols = useSymbols || b
+	}
+
+	useSymbols = useSymbols || cmd.useSymbols
```

Comment:
How does this work when the `--symbols` flag is set to `false`, but the `SECRETHUB_GENERATE_RAND_SYMBOLS` envar is set to `true`?

Reason I ask is twofold:
- First, the multiple conditionals make it a bit hard to reason about it so I had to read it a few times to understand what's happening. 
- Second, it seems to me like it may not follow the order of precedence. IMO the order of precedence should be:
   1. `--symbols` flag 
   2. `SECRETHUB_GENERATE_SYMBOLS` environment variable
   3. `SECRETHUB_GENERATE_RAND_SYMBOLS` environment variable